aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -6,6 +6,7 @@ module aptos_framework::fungible_asset {
     use aptos_framework::event;
```

Comment:
should we have a FungibleAssetManagementPermission - i.e. gating creating fungible assets and modifying their metadata, etc.

not sure if freezing should be part of the same, or separate one

Reply:
That's not needed thanks to the FA design. Those are already encoded as permission objects.